src/lkl/setup.c

```diff
@@ -625,6 +625,25 @@ void lkl_poststart_net(enclave_config_t* encl, int net_dev_id) {
             }
         }
 
+        res = lkl_if_set_ipv6(ifidx, encl->net_ip6.s6_addr, encl->net_mask6);
```

Comment:
We have to set the ipv6 address after bringing the interface up because lkl performs duplicate address detection when applying an address.